Core/src/Surfaces/Surface.cpp

```diff
@@ -254,14 +254,10 @@ Acts::RotationMatrix3 Acts::Surface::referenceFrame(
 }
 
 Acts::BoundToFreeMatrix Acts::Surface::boundToFreeJacobian(
-    const GeometryContext& gctx, const BoundVector& boundParams) const {
-  // Transform from bound to free parameters
-  FreeVector freeParams =
```

Comment:
Oh ok, so we would convert free to bound, only to then convert back to free, right?

Reply:
Right, I'm just saying that prior to this PR, we would most often have free params, convert them to bound, call this function, and then convert back to free. With this PR we don't anymore.